### PR TITLE
Dashboard with components set up

### PR DESCRIPTION
**Pull request by Chrys-Nicolaides**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React, { useState } from "react";
-import LandingPage from "./components/landingPage";
+// import LandingPage from "./components/landingPage";
 import Navbar from "./components/navbar";
+import Dashboard from "./components/Dashboard";
+import { Switch, Route, BrowserRouter } from "react-router-dom";
+
 // import Requests from "./components/Requests";
```
Comment (cameronnorman) [inline src/App.jsx:8]: Please remove comments of imports if they are not being used

File: src/components/TransactionItem.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        // icon = "AiOutlineEuro";
+        break;
+      case "Other":
+        categoryClass = " bg-orange-200 text-orange-900";
+        // icon = "AiOutlineCreditCard";
+        icon = "AiOutlineShop";
+        break;
+      default:
+        break;
+    }
```
Comment (cameronnorman) [inline src/components/TransactionItem.jsx:43]: Another note on the comments here. If they are not being used then just remove them

**Comment by cameronnorman** [comment]
Looking good! 🏆 I think the only thing I may want to bring your attention to is that there is a difference between a **component** and a **page**. I think when working with your router its better direct users to **page** that contains the **components** that it needs. For example you would have a **Pages folder** with **Home.js**. Inside **Home.js** you would have the **dashboard component**.  The pages folder lives on the same level as your components folder.

I have just seen this practise used at companies I have worked at and to me it makes sense.

**Review by cameronnorman — APPROVED** [review]

File: src/App.jsx (hunk 1 above)
Comment (Chrys-Nicolaides) [reply src/App.jsx:8]: Unused comments removed!

File: src/components/TransactionItem.jsx (hunk 2 above)
Comment (Chrys-Nicolaides) [reply src/components/TransactionItem.jsx:43]: Noted, thanks! I added a few icon alternatives per expense category for design review. I will remove the extras once we're happy with the final choices 👍

**Pull request merged**